pkg/rke/k8s_rke_system_images.go

```diff
@@ -9420,6 +9420,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
+		// Enabled out of band post v2.7.10
+		"v1.25.16-rancher2-3": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.9",
+			Kubernetes:                "rancher/hyperkube:v1.25.16-rancher2",
+			Alpine:                    "rancher/rke-tools:v0.1.96",
+			NginxProxy:                "rancher/rke-tools:v0.1.96",
+			CertDownloader:            "rancher/rke-tools:v0.1.96",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.96",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.22.28",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.22.28",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.22.28",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.6",
+			Flannel:                   "rancher/mirrored-flannel-flannel:v0.21.4",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher8",
```

Comment:
These are updated compared to `v1.25.16-rancher2-2`. Any specific reason we need to have them bumped? Otherwise asking to keep it the same.
```suggestion
			Flannel:                   "rancher/mirrored-flannelcni-flannel:v0.19.2",
			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher7",
```

Reply:
Mistakenly pasted; fixed it.